data/indicator_3-4-2.csv

```diff
@@ -1,20 +1,24 @@
-Year,Value
-1999,10.5
-2000,10.4
-2001,10.7
-2002,11.0
-2003,10.9
-2004,11.1
-2005,11.0
-2006,11.2
-2007,11.5
-2008,11.8
-2009,12.0
-2010,12.4
-2011,12.7
-2012,12.9
-2013,13.0
-2014,13.4
-2015,13.8
-2016,13.9
-2017,14.5
+﻿Year,Value

```

Comment:
FYI your CSV's are all encoded with Byte Order Marks. Not sure if that's going to cause issues with the ingestion or if they'll just generate fine. 

@brockfanning any ideas if a BOM will cause issues?

<img width="1392" alt="utf-with-bom" src="https://github.com/user-attachments/assets/d4ae661e-fc63-4158-aca7-198d92350d59">

Reply:
I believe you can also just strip those out by changing the file type above.